src/hotspot/share/oops/compressedKlass.hpp

```diff
@@ -50,12 +113,10 @@ class CompressedKlassPointers : public AllStatic {
   static address _base;
   static int _shift;
 
-  // Together with base, this defines the address range within which Klass
-  //  structures will be located: [base, base+range). While the maximal
-  //  possible encoding range is 4|32G for shift 0|3, if we know beforehand
-  //  the expected range of Klass* pointers will be smaller, a platform
-  //  could use this info to optimize encoding.
-  static size_t _range;
+  // Start and end of the Klass Range.
+  // Note: guaranteed to be aligned to 1<<shift (klass_alignment_in_bytes)
```

Comment:
>1<<shift (klass_alignment_in_bytes)

This is confusing to me, you mean `1 << klass_alignment_in_bytes`, right?

Reply:
No, its correct. "1 << shift" is klass alignment, in bytes.